launcher/src/CmderLauncher.cpp

```diff
@@ -87,6 +87,7 @@ void StartCmder(std::wstring path)
 	wchar_t exeDir[MAX_PATH] = { 0 };
 	wchar_t icoPath[MAX_PATH] = { 0 };
 	wchar_t cfgPath[MAX_PATH] = { 0 };
+	//wchar_t cfgExpPath[MAX_PATH] = { 0 };
```

Comment:
Could you outright remove this rather than commenting it out, that's why source control is a thing.